docs/getting-started.rst

```diff
@@ -25,7 +25,7 @@ CB Response or CB Protection servers inside the API credential file and select w
 at runtime.
 
 To create the initial credential file, a simple-to-use script is provided. Just run the ``cbapi-response``,
-``cbapi-protection``, or ``cbapi-defense`` script with the ``configure`` argument. On Mac OS X and Linux::
+``cbapi-protection``,``cbapi-psc`` or ``cbapi-defense`` script with the ``configure`` argument. On Mac OS X and Linux::
```

Comment:
let's just remove `cbapi-defense` as part of this... the `cbapi.psc.defense` module should work with the `credentials.psc` file - double check this of course